assets/os2forms_payment.css

```diff
@@ -0,0 +1,7 @@
+#testHallo--description {
+  font-size: 20px !important;
+}
+
+.webform-element-type-os2forms-payment {
+  display: none;
```

Comment:
Is this really needed? Does the Payment element render any content we don't want to show?